engine/lite/embeddedDeploymentManager/src/main/scala/pl/touk/nussknacker/streaming/embedded/EmbeddedDeploymentManager.scala

```diff
@@ -24,7 +24,9 @@ import scala.util.{Failure, Success}
 class EmbeddedDeploymentManagerProvider extends DeploymentManagerProvider {
 
   override def createDeploymentManager(modelData: ModelData, engineConfig: Config)
-                                      (implicit ec: ExecutionContext, actorSystem: ActorSystem, sttpBackend: SttpBackend[Future, Nothing, NothingT]): DeploymentManager = {
+                                      (implicit ec: ExecutionContext, actorSystem: ActorSystem,
+                                       sttpBackend: SttpBackend[Future, Nothing, NothingT],
+                                       deploymentService: ProcessingTypeDeploymentService): DeploymentManager = {
```

Comment:
`DeploymentManger` handles scenarios of given type - it is how our domain looks like now. It looks quite naturally that `DeploymentManager` has access to service that is in this scenario type scope. I used `ProcessingType` instead of `ScenarioType` because it is used in code in other places. What other name u would like to see here? `DeploymentService`? And current `DeploymentService` would be `AllScenarioTypesDeploymentService`? :)